starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -33,5 +35,19 @@ public InventoryController(InventoryDAO inventoryDAO) {
   public List<Inventory> findInventories() {
     return this.inventoryDAO.findAll();
   }
+
+
+  /**
+   * Create an Inventory.
+   * @param inventory the inventory to be created.
+   * @return The created inventory.
+   * 
+   * This is my code!
```

Comment:
remove this part of the Java docs comment.
You will want to make sure that Java doc are meaningful